engine/engine_test.go

```diff
@@ -71,101 +72,78 @@ func TestNewCryptoEngine_Cmd(t *testing.T) {
 	c := pkg.CryptoInstance()
 	c.Config.Fspath = "../temp"
 	c.Configure()
+	c.GenerateKeyPairFor(types.LegalEntity{URI: "legalEntity"})
+	cmd := e.Cmd
 
 	t.Run("Cmd returns a command with a single subCommand", func(t *testing.T) {
-		cmd := e.Cmd
-		if cmd.Name() != "crypto" {
-			t.Errorf("Expected Cmd name to equal [crypto], got %s", cmd.Name())
-		}
-
-		if len(cmd.Commands()) != 3 {
-			t.Errorf("Expected Cmd to have 3 sub-command, got %d", len(cmd.Commands()))
-		}
+		assert.Equal(t, "crypto", cmd.Name())
+		assert.Len(t,  cmd.Commands(), 3)
 	})
 
 	t.Run("Running generateKeyPair with too few arguments gives error", func(t *testing.T) {
-		cmd := e.Cmd
-
 		cmd.SetArgs([]string{"generateKeyPair"})
-		cmd.SetOutput(new(bytes.Buffer))
+		cmd.SetOut(new(bytes.Buffer))
```

Comment:
I gather this is just a way to discard the output? I think `ioutil.Discard` would be a more idiomatic approach.

Reply:
replaced instances